File: lesson_14/practice/Find_element.js
function firstElement(arr, n, k) {
    // This loop is used for selection
    // of elements
    for (let i = 0; i < n; i++) {
        // Count how many times selected element
        // occurs
        let count = 0;
        for (let j = i; j < n; j++) {
            if (arr[i] === arr[j]) {
                count++;
            }
        }

        // Check, if it occurs k times or not
        if (count === k) {
            return arr[i];
        }
    }

    return -1;
}

let arr = [1, 7, 4, 3, 4, 8, 7];
    let n = arr.length;
    let k = 2;
    console.log(firstElement(arr, n, k));

File: lesson_14/practice/FindElement.java
package lesson_14.practice;

public class FindElement {
    public static void main(String[] args) {
        int[] arr = {1, 7, 4, 3, 4, 8, 7};
        int n = arr.length;
        int k = 2;
        System.out.print(firstElementNative(arr, n, k));

    }
    public static int firstElementNative(int[] arr, int n, int k) {
        // This loop is used for selection of elements
        for (int i = 0; i < n; i++) {
            int count = 0; // Count how many times selected element occurs
            for (int j = i; j < n; j++) {
                if (arr[i] == arr[j]) {
                    count++;
                }
            }
            if (count == k) { // Check, if it occurs k times or not
                return arr[i];
            }
        }
        return -1;
    }

   
}
